document/CoreFormat.mdk

```diff
@@ -649,6 +649,49 @@ If the operating system does not provide such a value, the `fileVersion` propert
 On the Windows platform, this information is available in the `FILEVERSION` member of the `VERSIONINFO` structure.
 ~
 
+
+### `language` property { #tool-language }
+
+A tool object should contain a property named `language`, whose value is a string
+specifying the language of the messages produced by the tool, in the format specified by
+RFC 3066.
+
+~ IsoNumberedExample { @IsoNumberedExample=1 }
+The tool language is English:
+```
+"tool": {
+  "language": "en"
+```
+~
+
+~ IsoNumberedExample
+The tool language is French as spoken in France:
+```
+"tool": {
+  "language": "fr-FR"
+```
+~
+
+### `sarifLoggerVersion` property { #tool-sarifLoggerVersion }
+
+If the tool that produced the log relied on another software component to generate the log,
+then the `tool` object should contain a property named `sarifLoggerVersion`, whose value is
+a string specifying the version of the logging component.
+
+If this property is present, then it shall contain the Semantic Version 2.0.0 version
```

Comment:
ok, just a nit, but here's a place I don't think we have to encourage people to use semver. The logging component version should be sufficient to uniquely identify the implementation that produced the log file. That's it. We don't need to encourage standardizing on any particular format because we don't expect mgmt systems to process this data in a uniform way, across multiple tools, etc. This information exists primarily for this scenario: someone has observed a bug in the SARIF output and needs to know what code base produced it.